demos/ripple.html

```diff
@@ -57,7 +57,14 @@
       }
 
       section {
-        margin: 48px;
+        align-items: center;
+        display: flex;
+
+      }
+
+      section > div {
+        margin: 48px 0;
+        width: 400px;
```

Comment:
Or just `max-width: 400px;` so on small screens it just gets smaller as-needed. But doesn't grow above that.